test/test_czi.py

```diff
@@ -0,0 +1,29 @@
+import shutil
+from pathlib import Path
+from em_workflows.file_path import FilePath
+
+
+def test_input_fname(mock_nfs_mount, caplog):
+    from em_workflows.czi.flow import flow
+
+    state = flow.run(
+        input_dir="test/input_files/IF_czi/Projects/smaller",
+        no_api=True,
+    )
+    assert state.is_successful()
+
+
+def test_rechunk(mock_nfs_mount, caplog):
+    from em_workflows.czi.flow import gen_imageSet
+
+    input_dir = Path(
+        "/gs1/home/hedwig_dev/image_portal_workflows/image_portal_workflows/test/input_files/IF_czi/Projects/zarr_dir_2/"
+    )
```

Comment:
Can you verify that this `input_dir` has multiple `czi` files, and reverify the expected output?

Reply:
https://gist.github.com/philipmac/03ffea85b4567b2699f3a4781cca7540

dups in assets